package/libs/openssl/Makefile

```diff
@@ -321,6 +322,39 @@ ifdef CONFIG_i386
   endif
 endif
 
+define Host/Configure
+	(cd $(HOST_BUILD_DIR); \
+		MACHINE="$(HOST_ARCH_UNAME)" \
+		./config \
+			--prefix=$(STAGING_DIR_HOSTPKG)/usr \
+			--libdir=lib \
+			--openssldir=$(STAGING_DIR_HOSTPKG)/etc/ssl \
+			$(HOST_CPPFLAGS) \
+			$(HOST_LDFLAGS) \
+			shared \
+			no-dynamic-engine \
+			no-engine \
```

Comment:
now that I have learned there is a valid reason to avoid shared objects sometimes, I need to see whether it works without them...